apps/content/services/admin/asset_recitation_ayah_timestamps_upload_service.py

```
@@ -0,0 +1,102 @@
+
+                if to_create:
+                    RecitationAyahTiming.objects.bulk_create(to_create, batch_size=2000)
+                if to_update:
+                    RecitationAyahTiming.objects.bulk_update(
+                        to_update, fields=["start_ms", "end_ms", "duration_ms"], batch_size=2000
+                    )
+
+                created_total += len(to_create)
+                updated_total += len(to_update)
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)